src/expressions/block-expr.md

```diff
@@ -4,19 +4,11 @@
 > _BlockExpression_ :\
 > &nbsp;&nbsp; `{`\
 > &nbsp;&nbsp; &nbsp;&nbsp; [_InnerAttribute_]<sup>\*</sup>\
-> &nbsp;&nbsp; &nbsp;&nbsp; _Statements_<sup>\*</sup>\
+> &nbsp;&nbsp; &nbsp;&nbsp; [_Statement_]<sup>\*</sup>\
```

Comment:
Just to be clear, are you saying that because block-expressions can be confused as to whether it is a statement or a tail expression?

This kinda gets back to my uneasiness with how statements are defined.  Technically they do not include the trailing semicolon, they are *separated* by semicolons.  I'm thinking about rewriting the grammar to express it that way. What do you think?

Reply:
yes `{ { 2 } }` has a tail expression, not a tail-statement.

If you can get statements without `;` to work well then I would prefer that but I couldn't work out how to do it myself.